### DISCUSSION TITLE
Change between light and dark theme dynamically

### DISCUSSION THREAD

**Discussion by jpgarza93** [jothepro/doxygen-awesome-css#23] [Q&A]
Is it possible to add a button to allow changing between light and dark themes dynamically?

Or is it possible for me to change the theme dynamically?

Best and thanks!

**Comment by jothepro** [comment]
No, this is currently not possible. The theme is bound to the browser preference and can not be overridden dynamically.
Pls see #2 for more details.

----

Update:

Manually changing the theme with a button is now possible with the [experimental dark mode toggle](https://github.com/jothepro/doxygen-awesome-css#dark-mode-toggle-experimental).

**Reply by jpgarza93** [reply]
Okay, thank you very much. 

So it changes themes based on the system's _(in my case Windows OS)_ theme and if it detects dark mode, it applies dark mode. Correct?

**Reply by jothepro** [reply]
Yes, that's correct, at least for now.

**Reply by jpgarza93** [reply]
Okay, thank you so much. You have done a fantastic job.

**Reply by jothepro** [reply]
FYI Things have changed and manually changing the theme with a button is now possible with the [experimental dark mode toggle](https://github.com/jothepro/doxygen-awesome-css#dark-mode-toggle-experimental).

**Comment by jothepro** [comment]
Switching between light/dark mode is now possible in both pure Doxygen and Doxygen-awesome-css. I am closing the discussion for now.
